scss/bootstrap/overrides/style/_panel.scss

```diff
@@ -1,8 +1,10 @@
 .panel {
   @include box-shadow($crm-panels-box-shadow);
 
+  border-radius: 2px;
   border: none;
   margin-bottom: $grid-gutter-width;
+  overflow: hidden;
```

Comment:
Border radius is not applied to panels without having overflow hidden. not sure if there is any better approach